site-src/guides/implementers.md

```diff
@@ -27,6 +27,13 @@ like APIs delivered in the core Kubernetes binaries, each time a release happens
 the APIs have been reviewed by upstream Kubernetes reviewers, just like the APIs
 delivered in the core binaries.
 
+### API Semantics
+
+Gateway API MUST follow the Kubernetes API semantics. 
+The concepts are documented [here](https://kubernetes.io/docs/reference/using-api/api-concepts/).
+[Server-Side Apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) should also be 
```

Comment:
This link doesn't really provide a lot of information about what a controller author needs to do to use SSA, do you happen to have any more detailed references we could link to as well? If not, can we include some examples here?